tests/foreman/ui/test_adusergroups.py

```diff
@@ -19,12 +25,21 @@ def test_create_ldap_auth_withad(self):
 
         @Assert: Whether creating LDAP Auth with AD is successful.
 
-        @Status: Manual
-
         """
-
-    @stubbed()
-    def test_delete_ldap_auth_withad(self):
+        with Session(self.browser) as session:
+            make_ldapauth(session, name=server_name, server=self.ldap_hostname,
+                          server_type=LDAP_SERVER_TYPE['ad'],
+                          login_name=LDAP_ATTR['login_ad'],
+                          first_name=LDAP_ATTR['firstname'],
+                          surname=LDAP_ATTR['surname'], mail=LDAP_ATTR['mail'],
+                          account_user=self.ldap_username,
+                          account_passwd=self.ldap_passwd,
+                          account_basedn=self.ldap_basedn,
+                          account_grpbasedn=self.ldap_grpbasedn)
```

Comment:
Can you just format this a bit differently?

```python
        with Session(self.browser) as session:
            make_ldapauth(
                session,
                name=server_name,
                server=self.ldap_hostname,
                server_type=LDAP_SERVER_TYPE['ad'],
                login_name=LDAP_ATTR['login_ad'],
                first_name=LDAP_ATTR['firstname'],
                surname=LDAP_ATTR['surname'],
                mail=LDAP_ATTR['mail'],
                account_user=self.ldap_username,
                account_passwd=self.ldap_passwd,
                account_basedn=self.ldap_basedn,
                account_grpbasedn=self.ldap_grpbasedn,
            )
```

I find it much easier to browse through the arguments in this form.